cli/src/commands/create-expo-stack.ts

```diff
@@ -119,8 +118,9 @@ const command: GluegunCommand = {
         await runIgnite(toolbox, cliResults.projectName, cliResults);
       } else {
         // Check if the user wants to not install dependencies and/or not initialize git, update cliResults accordingly
-        cliResults.flags.noInstall = options.noInstall || false;
-        cliResults.flags.noGit = options.noGit || false;
+        cliResults.flags.noInstall =
+          options.noInstall || (typeof options.install === 'boolean' && !options.install) || false;
+        cliResults.flags.noGit = options.noGit || (typeof options.git === 'boolean' && !options.git) || false;
```

Comment:
is `--git true` / `--git` and `--install true` / `--install` the idea here? I'm not sure this makes sense because the checks are irrelevant / the default anyway.

If you don't include the check, this will still resolve correctly if they have `--git`, right?